sagemaker-debugger/pytorch_iterative_model_pruning/iterative_model_pruning.ipynb

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line 33: 
print("Training Job ", estimator.latest_training_job.name, " finished.")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1057/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>This is still unresolved.</p>